o3de/o3de
# Fixed pyside2 standalone

Signed-off-by: Jonny Galloway <gallowj@amazon.com>

## What does this PR do?
1. fixes DCCsi allowing PySide2 to work standalone (external of Editor framework)
2.  added readme marker for moved blender addons

_Please add links to any issues, RFCs or other items that are relevant to this PR._
https://github.com/o3de/o3de/issues/12807

## How was this PR tested?
1. Run confiig.py from IDE, with CLI args:  -py=True -qt=True -tp=True -gd=True -dm=True
2. Open CMD, change directory to DccScriptingInterface 
3. Run from DCCsi >python config.py -py=True -qt=True -tp=True -gd=True -dm=True

Result: a standalone PySide2 test button should pop-up

![image](https://user-images.githubusercontent.com/23222931/205999544-719a46c8-1aba-470b-8d42-51c3959c1c98.png)
![image](https://user-images.githubusercontent.com/23222931/205999584-ad40a476-a65d-41b5-b076-a5fcf4bde074.png)
![image](https://user-images.githubusercontent.com/23222931/205999618-ae6a31cd-53ed-44e4-88af-255f1fad7b50.png)
